cvat/apps/analytics_report/report/primary_metrics/imetric.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from cvat.apps.analytics_report.report.primary_metrics.utils import make_clickhouse_query
+from datetime import datetime, timezone
+
+class IPrimaryMetric():
+    _title = None
+    _description = None
+    _query = None
+    _granularity = None
+    _default_view = None
+    _transformations = []
```

Comment:
Such definition will define class fields, not instance fields. I'm not sure if it was supposed.